src/main/java/net/dv8tion/jda/api/interactions/components/Button.java

```diff
@@ -68,6 +68,21 @@
  */
 public interface Button extends Component
 {
+    /**
+     * The maximum length a button label can have
+     */
+    int LABEL_MAX_LENGTH = 80;
```

Comment:
Add `@see` tags to all constants similar to how `MessageEmbed` does

Reply:
Unlike MessagEmbed, there is no builder or setter for buttons.
Imo linking to the copy constructors adds no value

Reply:
I think a more valuable link would be from `withLabel` to the constant, especially the throws descriptions currently still use the hardcoded 80 instead of the constant. You can use `{@value #LABEL_MAX_LENGTH}` to render the value of a constant in documentation.

Reply:
`@value` will render in the actual value, but should we instead stick with `@link` so that developers know that there is a variable that they can compare against? Seems like it might be more valuable

Reply:
Values are highlighted and I don't have to look in the variable definition to know which value is there.
Links, however, would be consistent with embeds.

Reply:
This exact wording isn't necessarily what we need, but I think this documentation is more explicit and thus better
https://github.com/DV8FromTheWorld/JDA/blob/d64de79db8c7a1581c4e9ab44472193d25327672/src/main/java/net/dv8tion/jda/api/entities/MessageEmbed.java#L381

It renders out as 
![image](https://user-images.githubusercontent.com/1479909/128204146-d40a7a20-2e99-493f-ad60-203e5870caa9.png)

That feels more explicit as to the fact that there is a variable you can rely on than 
![image](https://user-images.githubusercontent.com/1479909/128204245-8ab97e8f-7ee2-4417-bdb7-8dff37d0c352.png)

Reply:
I think I've caught all the spots now. While editing i unified the exception documentation. Ready to review again